ws-nextjs-app/pages/[service]/send/[id]/FormContext/index.tsx

```diff
@@ -78,17 +80,25 @@ export const FormContextProvider = ({
   const [submitted, setSubmitted] = useState(false);
   const [progress, setProgress] = useState('0');
   const [submissionError, setSubmissionError] = useState<SubmissionError>(null);
+  const [hasAttemptedSubmit, setAttemptedSubmit] = useState(false);
 
   const handleChange = (name: OnChangeInputName, value: OnChangeInputValue) => {
```

Comment:
There may be a slight issue running this on the `onChange` handlers of the inputs. For example, before submitting the form, if you start typing in the email address field it will show the validation error right away. I feel like this is something we want to avoid as we only want to validate on 'submission' of the form. The validation messages can be removed when an `onChange` event occurs, but the actual validation function itself should only run when the user clicks 'submit'.

This may be more linked to the upcoming error handling work we discussed though.

Reply:
Ello, would this fix it: https://github.com/bbc/simorgh/pull/11626/commits/6212173a781d968d4e1cce81453c2d6d991d6c1a

Reply:
It fixes the initial email validation appearing, however if the user submits an invalid form, then enters values into the email address field it will attempt to valid as the user types.